### DISCUSSION TITLE
关于番剧时间表

### DISCUSSION THREAD

**Discussion by s10106** [the1812/Bilibili-Evolved#2923] [功能建议 / Ideas]
能不能直接显示新番和国创的时间表啊？我觉得现在的番剧和国创的版块直接显示时间表就够了了。这个最近更新以及周一至周日的番剧显示完全是多余的，就显示一个时间表加上右边的排行，我觉得而这样就挺好。请问能做一个这样的设置吗？

![12321](https://user-images.githubusercontent.com/53890304/150749525-2b6ca7da-534a-45c9-a9dd-378132150bbc.jpg)
![232323](https://user-images.githubusercontent.com/53890304/150749661-dbd9e4c9-8de0-4227-80b8-77e2cbb641f0.jpg)

**Comment by the1812** [comment]
时间表只会在 `清爽首页` 功能中提供, 目前还未完成

**Reply by s10106** [reply]
好的，以后有这个功能就好
